prov/tcp/src/tcpx_ep.c

```diff
@@ -284,7 +284,7 @@ static void tcpx_ep_flush_all_queues(struct tcpx_ep *ep)
 	ofi_bsock_discard(&ep->bsock);
 }
 
-void tcpx_ep_disable(struct tcpx_ep *ep, int cm_err)
+void tcpx_ep_disable(struct tcpx_ep *ep, int cm_err, void* err_data, size_t err_data_size)
```

Comment:
Check line length here and in header file

Reply:
I base splitting on the line focused on keeping it readable.  For example, I might do:
```
static struct ofi_some_structure *
foo(struct blah *blahblah, int param2, size_t param3, ...)
```
if the function type is long.  But if the function type is short (void, int, etc.) I'd wrap on one of the params, maybe even based on what the param itself is.
```
void tcpx_ep_disable(struct tcpx_ep *ep, int cm_err,
        void *err_data, size_t err_data_size);
```
except aligned better than I can do in this markdown.  In this case, I would wrap to keep err_data and err_data_size together, since they're treated as a pair.  But honestly splitting after err_data would be fine.  It doesn't help to add a break after the void as that doesn't impact the line length enough.